src/test/file-utils.test.js

```diff
@@ -0,0 +1,55 @@
+const assert = require('assert');
+const fileUtils = require('../main/lib/file-utils');
+
+describe('file-utils', () => {
+
+  describe('listFilesInDirectory', () => {
+    it('lists all files in the directory if no filters are provided', async () => {
```

Comment:
Please remove async, it is not needed here. Same in other test cases.

Reply:
Done.